tests/integration/security/sds_vault_flow/sds_vault_flow_test.go

```diff
@@ -37,6 +37,9 @@ func TestSdsVaultCaFlow(t *testing.T) {
 	framework.NewTest(t).
 		RequiresEnvironment(environment.Kube).
 		Run(func(ctx framework.TestContext) {
+			// Istio 1.3 uses Trustworthy JWT, which, unlike normal k8s JWT, is not
+			// recognized on Vault.
+			t.Skip("skipped for Istio 1.3, which uses Trustworthy JWT")
```

Comment:
Do you have a bug number for tracking when vault supports trustworthy jwt? So that we can take action items to re-enable Istio support for Vault.

And add the issue number here, in the comments, thanks.

Reply:
+1 all skips should have a linked issue

Reply:
Done: issues added.

Reply:
indent seems weird, also put the second one in the skip() message?

Reply:
if you are fixing that remove "istio 1.3". This is going to master which is 1.4

Reply:
Done.